msal_extensions/libsecret.py

```diff
@@ -15,16 +15,32 @@
 """
 import logging
 
-import gi  # https://pygobject.readthedocs.io/en/latest/getting_started.html
-
-# pylint: disable=no-name-in-module
-gi.require_version("Secret", "1")  # Would require a package gir1.2-secret-1
-# pylint: disable=wrong-import-position
-from gi.repository import Secret  # Would require a package gir1.2-secret-1
-
-
 logger = logging.getLogger(__name__)
 
+try:
+    import gi  # https://pygobject.readthedocs.io/en/latest/getting_started.html
+except ImportError:
+    logger.exception(
```

Comment:
That would not be possible this time, since we've demoted the `build_persistence(..., fallback_to_plaintext=False)` helper to become a sample only. :-)